src/mmrelay/matrix/room_mapping.py

```
@@ -0,0 +1,365 @@
+        for room_id, room in room_list:
+            room_name = getattr(room, "display_name", room_id)
+            encrypted = getattr(room, "encrypted", False)
+
+            # Format with encryption status
+            if e2ee_status["overall_status"] == "ready":
+                if encrypted:
+                    facade.logger.info(f"    🔒 {room_name}")
+                else:
+                    facade.logger.info(f"    ✅ {room_name}")
+            else:
+                if encrypted:
```

Ruff (0.15.9)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)